src/main/content/_posts/2017-12-21-jsf-implementation-open-liberty-17004.adoc

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title:  "Bring your own JSF implementation to Open Liberty 17.0.0.4"
+date:   2017-12-21 12:05:00 +0100
+categories: news
+author_picture: https://avatars3.githubusercontent.com/u/3322532
+---
+:description: Download updates for JSF Container (bring your own JSF implementation!), Concurrency, and distributed tracing in Open Liberty 17.0.0.4.
+= Bring your own JSF implementation to Open Liberty 17.0.0.4
+Laura Cowen <http://github.com/lauracowen>
+
+And, like a flash, a second release of Open Liberty (17.0.0.4) is upon us! Fancy bringing your own JSF implementation (Mojarra or MyFaces) to Open Liberty? You can now (and benefit from CDI) with the JSF Container 2.2 feature. Also, administrators can now configure concurrency policies for managed executors (Concurrency updates), and get distributed tracing with our implementation of opentracing.io.
```

Comment:
This sounds right, but to be honest I'm not familiar with the full system at work here. If you've done the diligence to make sure this is the function of the bring-your-own JSF implementation then it's fine.